api/controllers/UserController.js

```diff
@@ -99,33 +19,36 @@ module.exports = {
     res.locals.flash = {};
   },
 
+  // route initiates github authentication process
   auth: function(req, res) {
```

Comment:
NOT for this pull request, but we need to consider making this more explicit (e.g. authenticate vs. auth).